packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -541,7 +541,8 @@ class _RenderSegmentedControl<T> extends RenderBox
     RenderBox child = firstChild;
     double minWidth = 0.0;
     while (child != null) {
-      final _SegmentedControlContainerBoxParentData childParentData = child.parentData;
+      final _SegmentedControlContainerBoxParentData childParentData =
+        child.parentData as _SegmentedControlContainerBoxParentData;
```

Comment:
Here and elsewhere: these should still be one line from my understanding of the Flutter style guide.

Reply:
no problem to put it on one line. I usually split when the line length is more that 120 cols.

Reply:
That makes sense, but there's a preference to avoid linebreaks for assignments (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters)

Reply:
Done